flusher/flusher/init.py

```diff
@@ -94,3 +94,31 @@ def init(chain_id, topic, db):
             GROUP BY oracle_scripts.id, requests.resolve_status;
         """
     )
+    engine.execute(
+        """
+        CREATE VIEW non_validator_vote_proposals_view AS
+        SELECT
+        validator_id,
+        proposal_id,
+        answer,
+        CAST(SUM(shares) AS DECIMAL) * CAST(tokens AS DECIMAL) / CAST(delegator_shares AS DECIMAL) as amount
+        FROM delegations
+        JOIN votes ON delegations.delegator_id=votes.voter_id
+        JOIN validators ON delegations.validator_id=validators.id AND votes.voter_id != validators.account_id
+        JOIN accounts ON accounts.id=delegations.delegator_id
+        GROUP BY answer, validator_id , proposal_id, delegator_shares, tokens;"""
+    )
+    engine.execute(
+        """
+        CREATE VIEW validator_vote_proposals_view AS
```

Comment:
What did we want from this view? We want all tokens that delegate on this validator or we want what validator delegates themself?